api/api.go

```diff
@@ -39,6 +39,7 @@ import (
 	"github.com/iotexproject/iotex-core/action/protocol/poll"
 	"github.com/iotexproject/iotex-core/action/protocol/rolldpos"
 	"github.com/iotexproject/iotex-core/actpool"
+	l "github.com/iotexproject/iotex-core/api/logfilter"
```

Comment:
l is more like a local var name instead of a package name, consider use the full `logfilter` instead

Reply:
yes, I addressed it.